Mlem/Custom Tab Bar/FancyTabBar.swift

```diff
@@ -46,6 +46,14 @@ struct FancyTabBar<Selection: FancyTabBarSelection, Content: View>: View {
             }
     }
     
+    private func getAccessibilityLabel(tabIndex: Selection) -> String {
+        if let tabLabel = tabItems[tabIndex]?.tag.labelText {
+            return "\(tabLabel), Tab \(tabIndex.index) of \(tabItems.count.description)"
+        } else {
+            return "Tab \(tabIndex.index) of \(tabItems.count.description)"
+        }
+    }
+    
```

Comment:
This should also announce "Selected" at the beginning if the tab is the currently active one. As far as I can tell the structure is:

`[if selected: Selected], [if tabname: Tabname], Tab, [index of total]`

Reply:
Done!